### ISSUE TITLE
Linux arm64 release

### ISSUE THREAD

**Issue by chromer030** [BrowserWorks/waterfox#2766]
**Is your feature request related to a problem? Please describe.**
No.

**Describe the solution you'd like**
release binary for Linux arm64.

**Comment by defencedog** [comment]
I would like that as well especially with the rise of SBC popularity

**Comment by jaredcwhite** [comment]
As a recent convert to Asahi Linux running on Apple hardware…yes please! I like using Vivaldi, but I'd also like to be able to use this fantastic browser which I can do in macOS!

**Comment by MrAlex94** [comment]
Yes, just re-working on the website (because it'll be how we offer all the various options) and after that, my main priority is:

- ARM64 builds for Linux/Windows
- deb/rpm/flatpaks
- msix installer for windows

**Comment by b4rgut** [comment]
I use Waterfox as my daily driver on Fedora Linux (x86) and absolutely love it. Everything works perfectly. However, I recently set up a Mac Mini M2 running Fedora Asahi Remix (aarch64). Since there is no native ARM64 build of Waterfox for Linux, I am currently forced to fall back to regular Firefox on this machine.

With the active development of Asahi Linux on Apple Silicon and the rise of new ARM-based laptops (like Snapdragon X series), having an official aarch64 build for Linux would be incredibly helpful. I would love to keep my workflow consistent and use my favorite browser across all my devices.

**Comment by FoxyDrew** [comment]
> Yes, just re-working on the website (because it'll be how we offer all the various options) and after that, my main priority is:
> 
> * ARM64 builds for Linux/Windows
> * deb/rpm/flatpaks
> * msix installer for windows

Any updates on this since the last 6 months?

**Comment by MrAlex94** [comment]
https://download.opensuse.org/repositories/isv:/BrowserWorks/

Instructions at https://www.waterfox.com/download/ > Beta
